unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/Tests/UiContainerHandlerShould.cs

```diff
@@ -0,0 +1,73 @@
+using DCL.ECS7.InternalComponents;
+using DCL.ECSRuntime;
+using NUnit.Framework;
+using UnityEngine.UIElements;
+
+namespace Tests
```

Comment:
a more specific namespace would be better, for example: `DCL.ECSComponents`

Reply:
The main issue of using a namespace scheme like that for tests is that on unity editor will look like:
```
-> Assambly name
--> DCL
---> ECSComponents
----> Test class name
-----> Test method names
-----> [...] 
```
which makes super hard and a total waste of time when you need to manually select which tests to run, specially when different tests must be selected at the same time